zerver/lib/markdown/__init__.py

```diff
@@ -601,6 +602,32 @@ def run(self, root: Element) -> None:
             img.set("src", get_camo_url(url))
 
 
+class InlineVideoProcessor(markdown.treeprocessors.Treeprocessor):
+    """
+    Rewrite inline video tags to serve external content via Camo.
+
+    This rewrites all video, except ones that are served from the current
+    realm or global STATIC_URL. This is to ensure that each realm only loads
```

Comment:
Do we need to support the global STATIC_URL at all?

Reply:
Since Zulip is open source, someone might add videos to their static directory and would like to link them ? We allow it for image, so I just followed the pattern for it.